packages/quantic/force-app/main/default/lwc/quanticSearchInterface/quanticSearchInterface.js

```diff
@@ -68,7 +69,8 @@ export default class QuanticSearchInterface extends LightningElement {
   initUrlManager(engine) {
     this.urlManager = CoveoHeadless.buildUrlManager(engine, {initialState: {fragment: this.fragment}});
     this.unsubscribeUrlManager = this.urlManager.subscribe(() => this.updateHash());
-    window.addEventListener('hashchange', this.onHashChange);
+    this._onHashChange = this.onHashChange.bind(this);
+    window.addEventListener('hashchange', this._onHashChange);
```

Comment:
You could try changing the onHashChange from a method to a private property
```
onHashChange = () => {
  this.urlManager.synchronize(this.fragment);
};
```

Reply:
Yeah that's better. Thanks! Still have another issue i need to fix in community though